content/docs/command-reference/plots/index.md

```diff
@@ -49,6 +49,15 @@ Data-series plots utilize [Vega-Lite](https://vega.github.io/vega-lite/) for
 rendering (declarative JSON grammar for defining graphics). Image-type plots are
 rendered using `<img>` tags directly.
 
+<admon icon="fire">
+
+**New!** DVC Plots are available [inside the VS Code IDE], a quick and intuitive
+UI for DVC Experiments.
+
+[inside the vs code ide]: /doc/vs-code-extension
```

Comment:
Should we link directly to https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md in this case?